src/Router.js

```diff
@@ -1,13 +1,32 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useState, useEffect } from 'react';
 
 import Main from './pages/Main/Main';
+import Nav from './components/Nav/Nav';
 
 export default function Router() {
+  const [windowY, setWindowY] = useState();
+  const [visibleValue, setVisibleValue] = useState(true);
+
+  useEffect(() => {
+    function handleScroll() {
+      windowY > window.scrollY ? setVisibleValue(true) : setVisibleValue(false);
+      setWindowY(window.scrollY);
+    }
+    window.addEventListener('scroll', handleScroll);
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, [windowY, visibleValue]);
+
   return (
-    <BrowserRouter>
-      <Routes>
-        <Route path="/" element={<Main />} />
-      </Routes>
-    </BrowserRouter>
+    <div className="headContainer">
+      <Nav visibleValue={visibleValue} />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<Main />} />
+        </Routes>
+      </BrowserRouter>
+    </div>
```

Comment:
```suggestion      
      <BrowserRouter>
        <Nav visibleValue={visibleValue} />
        <Routes>
          <Route path="/" element={<Main />} />
        </Routes>
      </BrowserRouter>
```
제가 제안드렸던건 이러한 형태였는데 확인 부탁드려요

Reply:
확인해보니 현재 작업 중인 Main branch에서도 headContainer의 유무가 크게 중요하지 않은 것 같습니다.

따라서 현재 Nav branch에서는 삭제했고 컴포넌트 위치 또한 제안해주신 것과 같이 변경되었습니다.